lib/core/AccountManager.js

```diff
@@ -44,7 +45,24 @@ exports.AccountManager = class AccountManager {
                       if (res.results.length > 0) {
                         return this.redashClient().resendUserInvitation(context.user.email);
                       }
-                      return res.results[0];
+                      
```

Comment:
Style: There's a trailing space on L48 and L56. If you're using VScode, you can install `Trailing Spaces` extension to highlight it (and it also has delete command).